tools/projmgr/src/ProjMgrYamlEmitter.cpp

```diff
@@ -721,6 +721,10 @@ bool ProjMgrYamlBase::WriteFile(YAML::Node& rootNode, const std::string& filenam
     }
   }
   else if (!CompareFile(filename, rootNode)) {
+    if (!allowUpdate) {
+      ProjMgrLogger::Error(filename, "file not allowed to be updated");
```

Comment:
Would it be possible to be more specific about the reasons why it cannot be updated (e.g. which pack/version is required vs updated) maybe in `verbose` mode?

Reply:
Is this required in this PR? Both `yarn` and `cargo` does not specify the difference.

Reply:
It is not required, it is just a suggestion, what is your opinion? Regardless of what `yarn` and `cargo` do, would it bring helpful information to the user?

Reply:
I suppose it might indicate why the CI build failed, but the user would still need to do the same step locally in order to update the `*.cbuild-pack.yml` file.
Can you please create an issue for this, as I believe it should not be part of this PR.

Reply:
Just for clarity my comment concerns the changes in lines 724-725 of [`ProjMgrYamlEmitter.cpp`](https://github.com/Open-CMSIS-Pack/devtools/pull/1225/files/e5eced6777ff4602b9a4b21a5b27a73403e8c868#diff-622f5ce2a380acef913dfa78872551356f6b1be48e286f33de05da865078e738R724-R725):
```
    if (!allowUpdate) {
      ProjMgrLogger::Error(filename, "file not allowed to be updated");
```
In other words I am asking whether it is meaningful to increase the verbosity of this error message that is being introduced in this PR so the question is specific to this PR.
Also it should be indifferent whether it runs on CI or locally.